tests/helper.py

```diff
@@ -1,9 +1,21 @@
 from errbot.backends.test import TestBot
+from errbot.plugin_info import PluginInfo
+from errbot.templating import add_plugin_templates_path
+from pathlib import Path
 
 
 def plugin_testbot(klass, loglevel, config=None):
     config = config if config else dict()
     testbot = TestBot(loglevel=loglevel, extra_config=config)
     testbot.start()
-    plug = testbot.bot.plugin_manager.instanciateElement(klass)
+    plugin_name = klass.__name__
+    plug = klass(testbot.bot, plugin_name)
     return plug, testbot
+
+
+def load_templates(plug_file_name):
```

Comment:
Same, do we even need this after `CoroboTestCase`?

Reply:
Nope